test/Microsoft.Dnx.DesignTimeHost.FunctionalTests/DthStartupTests.cs

```diff
@@ -309,19 +276,18 @@ public static IEnumerable<object[]> UnresolvedDependencyTestData
         }
 
         [Theory]
-        [MemberData(nameof(RuntimeComponents))]
-        public void DthNegative_BrokenProjectPathInLockFile_V1(string flavor, string os, string architecture)
+        [MemberData(nameof(DnxSdks))]
+        public void DthNegative_BrokenProjectPathInLockFile_V1(DnxSdk sdk)
         {
-            var runtimeHomePath = _fixture.GetRuntimeHomeDir(flavor, os, architecture);
             var testProject = _fixture.GetTestProjectPath("BrokenProjectPathSample");
 
             using (var disposableDir = new DisposableDir())
-            using (var server = DthTestServer.Create(runtimeHomePath, testProject))
+            using (var server = DthTestServer.Create(sdk, testProject))
             using (var client = new DthTestClient(server, 0))
             {
                 // copy the project to difference location so that the project path in its lock file is invalid
                 var targetPath = Path.Combine(disposableDir, "BrokenProjectPathSample");
-                TestUtils.CopyFolder(testProject, targetPath);
+                CommonTestUtils.TestUtils.CopyFolder(testProject, targetPath);
```

Comment:
Why are you using the copy folder from CommonTestUtils instead of the new infrastructure?